src/Microsoft.DotNet.Darc/src/Darc/Operations/SetGoalOperations.cs

```diff
@@ -0,0 +1,51 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.Darc.Helpers;
+using Microsoft.DotNet.Darc.Options;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Net;
+using System.Threading.Tasks;
+using System.Linq;
+using Microsoft.DotNet.Maestro.Client.Models;
+using Microsoft.DotNet.Maestro.Client;
+
+namespace Microsoft.DotNet.Darc.Operations
+{
+    internal class SetGoalOperation : Operation
+    {
+        private SetGoalCommandLineOptions _options;
+        public SetGoalOperation(SetGoalCommandLineOptions options)
+            : base(options)
+        {
+            _options = options;
+        }
+
+        /// <summary>
+        ///  Sets Goal in minutes for a definition in a Channel.
+        /// </summary>
+        /// <returns>Process exit code.</returns>
+        public override async Task<int> ExecuteAsync()
+        {
+            try
+            {
+                IRemote remote = RemoteFactory.GetBarOnlyRemote(_options, Logger);
+                Goal goalInfo = await remote.SetGoalAsync(_options.Channel, _options.DefinitionId, _options.Minutes);
+                return Constants.SuccessCode;
```

Comment:
Should we output something about "goal set" or something, maybe outputting some of the values from the "goalInfo" object for the users information?

Reply:
according to the document -> https://github.com/dotnet/arcade/blob/master/Documentation/Darc.md , I must print out the goal in minutes.  I will print out the minutes :) thanks Chad!